sdk/main/include/FileCreateTransaction.h

```diff
@@ -17,236 +17,179 @@
+   * @throws UninitializedException If the input client has no operator with which to sign this
+   *                                FileCreateTransaction.
    */
-  inline InitType<KeyList> getKeys() const { return mKeys; }
+  [[nodiscard]] proto::Transaction makeRequest(const Client& client,
+                                               const std::shared_ptr<internal::Node>& /*node*/) const override;
 
   /**
-   * Extract the file contents.
+   * Derived from Executable. Submit this FileCreateTransaction to a Node.
    *
-   * @return The file contents.
+   * @param client   The Client submitting this FileCreateTransaction.
+   * @param deadline The deadline for submitting this FileCreateTransaction.
+   * @param node     Pointer to the Node to which this FileCreateTransaction should be submitted.
+   * @param response Pointer to the TransactionResponse protobuf object that gRPC should populate with the response
+   *                 information from the gRPC server.
+   * @return The gRPC status of the submission.
    */
-  inline std::string getContents() const { return mContents; }
+  [[nodiscard]] grpc::Status submitRequest(const Client& client,
+                                           const std::chrono::system_clock::time_point& deadline,
+                                           const std::shared_ptr<internal::Node>& node,
+                                           proto::TransactionResponse* response) const override;
 
   /**
-   * Extract the file's memo.
+   * Build a CryptoCreateTransactionBody protobuf object from this FileCreateTransaction object.
    *
-   * @return The file's memo.
-   */
-  inline std::string getFileMemo() const { return mMemo; }
-
-private:
-  /**
-   * Initialize from the transaction body.
+   * @return A pointer to a CryptoCreateTransactionBody protobuf object filled with this FileCreateTransaction object's
+   *         data.
    */
-  void initFromTransactionBody();
+  [[nodiscard]] proto::FileCreateTransactionBody* build() const;
 
   /**
-   * The time at which this file should expire (unless FileUpdateTransaction is
-   * used before then to extend its life)
+   * The time at which the new file will expire.
    */
-  InitType<std::chrono::nanoseconds> mExpirationTime;
+  std::chrono::system_clock::time_point mExpirationTime = std::chrono::system_clock::now();
 
   /**
-   * All keys at the top level of a key list must sign to create or modify the
-   * file. Any one of the keys at the top level key list can sign to delete the
-   * file.
+   * The key that must sign Transactions to mutate the new file.
    */
-  InitType<KeyList> mKeys;
+  std::shared_ptr<PublicKey> mKey = nullptr;
 
   /**
-   * The bytes that are the contents of the file.
+   * The contents of the new file.
    */
-  std::string mContents;
+  std::vector<std::byte> mContents;
 
   /**
-   * The memo associated with the file (UTF-8 encoding max 100 bytes).
+   * The memo for the new file.
    */
   std::string mMemo;
```

Comment:
I saw the Java version of this file ([FileCreateTransaction.java](https://github.com/hashgraph/hedera-sdk-java/blob/9467f0eaa50300048748ac6b57a6007be75013ac/sdk/src/main/java/com/hedera/hashgraph/sdk/FileCreateTransaction.java)) is using another naming for this field - [fileMemo](https://github.com/hashgraph/hedera-sdk-java/blob/9467f0eaa50300048748ac6b57a6007be75013ac/sdk/src/main/java/com/hedera/hashgraph/sdk/FileCreateTransaction.java#L53). Also the `getter` & `setter` methods could be renamed too.

It's not something critical, but I think it could be better to keep the fields naming the same between the `Java` & `C++ SDKs`.

Reply:
Fixed